packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -3,10 +3,163 @@ id: Release highlights
 title: Release highlights
 section: get-started
 ---
-
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>March 2024</Timestamp>
+
+## PatternFly 6.2
+
+### Promoted package versions
+- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/v6.2.0))
+    - [@patternfly/patternfly@6.2.0](https://www.npmjs.com/package/@patternfly/patternfly)
+- patternfly/react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/v6.2.0))
+- PatternFly extensions 
+    - [@patternfly/chatbot@2.2.0](https://www.npmjs.com/package/@patternfly/chatbot) ([changelog](https://github.com/patternfly/chatbot/releases/tag/v2.2.0))
+
+
+### Website updates
+
+##### Bugs 
+- We adjusted the SVG download behavior for [PatternFly icons](/design-foundations/icons/#all-icons), to remove a wrapping `<span>` that prevented imports into some design tools.
+- We improved accessibility for the hover text of PatternFly logos in the website masthead and footer.
+- We fixed an issue across our demos where the PatternFly masthead logo was the wrong color. 
+- We removed the Design with Sketch page, since that information is no longer current. References to the old PatternFly Sketch library now point to the v5 archive site.
+- Charts: 
+    - We updated the [alerts timeline example](/charts/bar-chart#alerts-timeline) to center tooltips above their respective bar.
+    - We fixed a few issues in CodeSandbox chart examples, including adding missing Victory dependency, adding missing `ChartThemeColor` import, ensuring Victory packages are up to date.
+- [Clipboard copy](/components/clipboard-copy): We added a missing action tooltip in the [inline compact with additional action](/clipboard-copy/#inline-compact-with-additional-action) example.
+- [Tabs:](/components/tabs) 
+    - We added a missing [props table for `TabContentBody`](/components/tabs#tabcontentbody).
+    - We removed the independent tab content page and moved the content within [the tabs HTML documentation](/components/tabs#tab-content). 
+
+##### Features
+- We expanded the [PatternFly 6 upgrade guide](/get-started/upgrade) to include additional instructions that we've put together as products have been completing their migration.
+
+### Component updates
+
+#### Bugs 
+- [Button](/components/button): We updated the `aria-disabled` implementation so that it will only render when `true`. If it's `false` or `undefined`, it will not render on the element.
```

Comment:
Might be worth also calling out that if a consumer has tests checking for aria-disabled to be false or to match the disabled property, those tests should be updated.